### PR TITLE
Add DappTools and Foundry starter kits

### PR DESCRIPTION
**Pull request by aelmanaa**
fixes https://github.com/smartcontractkit/documentation/issues/654

### CODE REVIEW

**Review by ritika-b — CHANGES_REQUESTED** [review]

File: docs/Developer Reference/create-a-chainlinked-project.md [hunk 1]
```diff
@@ -89,11 +89,43 @@ cd MyChainlinkProject/
...
+
+```shell
+git clone https://github.com/smartcontractkit/dapptools-starter-kit
+cd dapptools-starter-kit
+make # This installs the project's dependencies.
+```
+
+For instructions on how to use the starter kit, refer to the DappTools starter kit [README](https://github.com/smartcontractkit/dapptools-starter-kit#readme).
+
+For more details on how to use Chainlink with DappTools, see our blog post [How To Use DappTools](https://blog.chain.link/how-to-use-dapptools/).
```
Comment (dwightjl) [inline docs/Developer Reference/create-a-chainlinked-project.md:111]:
```suggestion
## DappTools Starter Kit

To learn more about DappTools, refer to the [DappTools Documentation](https://dapp.tools/).

1. Install Dapp tools using the [Installation instructions](https://github.com/dapphub/dapptools#installation) in the Dapp tools GitHub repository.

1. After you install the tools, clone the starter kit and install the dependencies:

      ```shell
      git clone https://github.com/smartcontractkit/dapptools-starter-kit
      cd dapptools-starter-kit
      make # This installs the project's dependencies.
      ```

For instructions on how to use the starter kit, refer to the DappTools starter kit [README](https://github.com/smartcontractkit/dapptools-starter-kit#readme).

For more details on how to use Chainlink with DappTools, see our blog post [How To Use DappTools](https://blog.chain.link/how-to-use-dapptools/).
```

File: docs/Developer Reference/create-a-chainlinked-project.md [hunk 2]
```diff
@@ -89,11 +89,43 @@ cd MyChainlinkProject/
...
+
+Once installed, clone the starter kit and install the project's dependencies:
+
+```shell
+git clone https://github.com/smartcontractkit/foundry-starter-kit
+cd foundry-starter-kit
+make # This installs the project's dependencies.
+```
+
+For instructions on how to use the starter kit, refer to the [Foundry starter kit README](https://github.com/smartcontractkit/foundry-starter-kit#readme).
```
Comment (dwightjl) [inline docs/Developer Reference/create-a-chainlinked-project.md:126]:
```suggestion
To learn more about Foundry, refer to the [Foundry Documentation](https://onbjerg.github.io/foundry-book/).

1. Install Foundry using the [Installation instructions](https://onbjerg.github.io/foundry-book/getting-started/installation.html) on GitHub.io.

1. After you install Foundry, clone the starter kit and install the projects dependencies:

      ```shell
      git clone https://github.com/smartcontractkit/foundry-starter-kit
      cd foundry-starter-kit
      make # This installs the project's dependencies.
      ```

For instructions on how to use the starter kit, refer to the [Foundry starter kit README](https://github.com/smartcontractkit/foundry-starter-kit#readme).
```

File: docs/Developer Reference/create-a-chainlinked-project.md [hunk 3]
```diff
@@ -89,11 +89,47 @@ cd MyChainlinkProject/
...
+For more details on how to use Chainlink with DappTools, see our blog post [How To Use DappTools](https://blog.chain.link/how-to-use-dapptools/).
+___
+
+## Foundry Starter Kit
+
+To learn more about Foundry, refer to the [Foundry Documentation](https://onbjerg.github.io/foundry-book/).
+
+1. Install Foundry using the [Installation instructions](https://onbjerg.github.io/foundry-book/getting-started/installation.html) on GitHub.io.
+
+1. After you install Foundry, clone the starter kit and install the projects dependencies:
```
Comment (dwightjl) [inline docs/Developer Reference/create-a-chainlinked-project.md:122]:
```suggestion
1. After you install Foundry, clone the starter kit and install the project dependencies:
```

File: docs/Developer Reference/create-a-chainlinked-project.md [hunk 4]
```diff
@@ -51,9 +51,9 @@ git clone https://github.com/smartcontractkit/hardhat-starter-kit
 cd hardhat-starter-kit
 ```
 
-For instructions about how to use the starter kit, see the [Hardhat starter kit README](https://github.com/smartcontractkit/hardhat-starter-kit/blob/main/README.md).
+For instructions on how to use the starter kit, refer to the [Hardhat starter kit README](https://github.com/smartcontractkit/hardhat-starter-kit/blob/main/README.md).
 
-For more information on how to use Chainlink with Hardhat, see our blog post [How to use Hardhat with Chainlink](https://blog.chain.link/using-chainlink-with-hardhat/).
+For more details on how to use Chainlink with Hardhat, see our blog post [How to use Hardhat with Chainlink](https://blog.chain.link/using-chainlink-with-hardhat/).
```
Comment (dwightjl) [inline docs/Developer Reference/create-a-chainlinked-project.md:56]:
```suggestion
For more details on how to use Chainlink with Hardhat, see our blog post about [How to use Hardhat with Chainlink](https://blog.chain.link/using-chainlink-with-hardhat/).
```

File: docs/Developer Reference/create-a-chainlinked-project.md [hunk 5]
```diff
@@ -68,9 +68,9 @@ cd MyChainlinkProject/
 brownie bake chainlink-mix
 cd chainlink-mix
 ```
-For instructions about how to use the starter kit, see the [Brownie starter kit README](https://github.com/smartcontractkit/chainlink-mix/blob/master/README.md).
+For instructions on how to use the starter kit, refer to the [Brownie starter kit README](https://github.com/smartcontractkit/chainlink-mix/blob/master/README.md).
 
-For more information on how to use Chainlink with Brownie, see our blog post [Develop a DeFi Project Using Python](https://blog.chain.link/develop-python-defi-project/).
+For more details on how to use Chainlink with Brownie, see our blog post [Develop a DeFi Project Using Python](https://blog.chain.link/develop-python-defi-project/).
```
Comment (dwightjl) [inline docs/Developer Reference/create-a-chainlinked-project.md:73]:
```suggestion
For more details on how to use Chainlink with Brownie, see the [Develop a DeFi Project Using Python](https://blog.chain.link/develop-python-defi-project/) blog post.
```

File: docs/Developer Reference/create-a-chainlinked-project.md [hunk 6]
```diff
@@ -89,11 +89,47 @@ cd MyChainlinkProject/
 truffle unbox smartcontractkit/truffle-starter-kit
 ```
 
-For instructions about how to use the starter kit, see the [Truffle starter kit README](https://github.com/smartcontractkit/truffle-starter-kit/blob/master/README.md).
+For instructions on how to use the starter kit, refer to the [Truffle starter kit README](https://github.com/smartcontractkit/truffle-starter-kit/blob/master/README.md).
 
 For more details on how to use Chainlink with Truffle, see our blog post [Using Truffle to interact with Chainlink Smart Contracts](https://www.trufflesuite.com/blog/using-truffle-to-interact-with-chainlink-smart-contracts).
```
Comment (dwightjl) [inline docs/Developer Reference/create-a-chainlinked-project.md:94]:
```suggestion
For more details on how to use Chainlink with Truffle, see our blog post about [Using Truffle to interact with Chainlink Smart Contracts](https://www.trufflesuite.com/blog/using-truffle-to-interact-with-chainlink-smart-contracts).
```

File: docs/Developer Reference/create-a-chainlinked-project.md [hunk 7]
```diff
@@ -89,11 +89,47 @@ cd MyChainlinkProject/
...
+
+      ```shell
+      git clone https://github.com/smartcontractkit/dapptools-starter-kit
+      cd dapptools-starter-kit
+      make # This installs the project's dependencies.
+      ```
+
+For instructions on how to use the starter kit, refer to the DappTools starter kit [README](https://github.com/smartcontractkit/dapptools-starter-kit#readme).
+
+For more details on how to use Chainlink with DappTools, see our blog post [How To Use DappTools](https://blog.chain.link/how-to-use-dapptools/).
```
Comment (dwightjl) [inline docs/Developer Reference/create-a-chainlinked-project.md:113]:
```suggestion
For more details on how to use Chainlink with DappTools, see the [How To Use DappTools](https://blog.chain.link/how-to-use-dapptools/) blog post.
```

**Review by dwightjl — COMMENTED** [review]
Minor sentence edits

**Review by dwightjl — APPROVED** [review]

**Pull request merged**